app/Providers/RepositoryServiceProvider.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+namespace App\Providers;
+
+use Illuminate\Support\ServiceProvider;
+
+class RepositoryServiceProvider extends ServiceProvider
+{
+    protected static $repositories = [
+        [
+            'App\Repositories\Contracts\OfficeRepository',
+            'App\Repositories\Eloquents\OfficeEloquentRepository',
+        ], 
+        [
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.SuperfluousWhitespace.EndLine</br>**Severity**: error</br>**File**: app/Providers/RepositoryServiceProvider.php</br>Whitespace found at end of line